app/src/main/java/org/fossasia/openevent/general/di/Scopes.kt

```diff
@@ -0,0 +1,13 @@
+package org.fossasia.openevent.general.di
+
+/**
+ * Enum class to collect all possible Fragment scopes for Koin DI
+ * in one place. This list is expected to grow as Scopes are used in more
+ * fragments.
+ */
+enum class Scopes(val scopeName: String) {
```

Comment:
No need of scopeName variable

Reply:
How can I associate a string value with an enum without this?

Reply:
toString(), name()?

Reply:
I can override the `toString()` method of each Enum but that is a lot more verbose and essentially serves the same purpose.

I mean something like this:
```
enum class Scopes {
  SCOPE_A {
    override fun toString(): String = "A"
  }
}
```

Reply:
Oh, okay. I'll override the toString() method then. Thanks.

Reply:
I've added this change now.

Reply:
@iamareebjamal Sorry for the mess. I had removed the scopeName variable but forgot to remove its usages in other parts of the codebase. I've fixed this now. Tested the app on my device, and everything works.